lib/pliny/log.rb

```diff
@@ -4,6 +4,10 @@ def log(data, &block)
       log_to_stream(stdout || $stdout, merge_log_contexts(data), &block)
     end
 
+    def log_without_context(data, &block)
```

Comment:
@gudmundur Yeah, thinking about it a little more, I agree with your original assessment. I added a change in #307.